lib/Conversion/TorchToTMTensor/TorchToTMTensor.cpp

```diff
@@ -52,6 +54,20 @@ using namespace mlir::torch::TMTensor;
 // that these patterns become mostly mechanical associations of
 // "aten.foo -> linalg.foo".
 
+static Attribute getNumericLimit(PatternRewriter& rewriter, Type elementType, bool getMin = true) {
+    auto bitWidth = elementType.getIntOrFloatBitWidth();
```

Comment:
Should be done

Reply:
This isn't right. The formatting should make the indentation two spaces rather than four. Depending on how you have your changes, you have to run `clang-format` differently. For example, if you have changes staged, `git clang-format` will only format the staged changes, and this will result as new unstaged changes. However, if your changes are already in a commit, then you need to do `git clang-format HEAD~1` to format the latest commit. This results in new changes that you then need to stage and commit. (You can also do multiple commits by changing `HEAD~1` to whatever range you need).

Reply:
Ah I think this is what I was missing, I will change the range and check it over

Reply:
I changed the range of commits and reformatted using clang-format which worked (sorry, my fault here). It should now all be formatted properly and two-spaced instead of four